modules/ROOT/pages/java/java-create-instance.adoc

```diff
@@ -8,27 +8,71 @@ endif::[]
 
 
 
-This example shows how to create an instance of a class:
+This example shows how to create an instance of a class whose constructor does not require parameters:
```

Comment:
There is no Person Class definition anywhere, this will help people understand how the java module works

Reply:
added java code